### PR TITLE
Update New York CityGML 3D Tileset

### PR DESCRIPTION
**Pull request by shehzan10**
* Better streaming performance with draco compressed tiles.
* Uses new tiling algorithm with smarter tiling.
* Longitude/Latitude are in degrees so there is a conversion in styling.
* Add lon/lat/height to the feature picking table.
* Remove the area styling - no longer a property of the tileset.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@shehzan10, thanks for the pull request! Maintainers, we have a signed CLA from @shehzan10, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Apps/Sandcastle/gallery/3D Tiles Feature Styling.html [hunk 1]
```diff
@@ -42,45 +42,41 @@
...
-    tileset.style = new Cesium.Cesium3DTileStyle({
-        color: "mix(color('yellow'), color('red'), min(${area} / 10000.0, 1.0))"
-    });
-}
-
 // Color buildings by their latitude coordinate.
 function colorByLatitude() {
     tileset.style = new Cesium.Cesium3DTileStyle({
+        defines: {
+            latitudeRadians: "Number(${latitude}) * 0.0174532925"
```
Comment (lilleyse) [inline Apps/Sandcastle/gallery/3D Tiles Feature Styling.html]: Use built in `radians` function: https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling#radians

File: Apps/Sandcastle/gallery/3D Tiles Feature Styling.html [hunk 2]
```diff
@@ -42,45 +42,41 @@
...
         color: {
             conditions: [
-                ["${height} >= 300", "rgba(45, 0, 75, 0.5)"],
-                ["${height} >= 200", "rgb(102, 71, 151)"],
-                ["${height} >= 100", "rgb(170, 162, 204)"],
-                ["${height} >= 50", "rgb(224, 226, 238)"],
-                ["${height} >= 25", "rgb(252, 230, 200)"],
-                ["${height} >= 10", "rgb(248, 176, 87)"],
-                ["${height} >= 5", "rgb(198, 106, 11)"],
+                ["Number(${height}) >= 300", "rgba(45, 0, 75, 0.5)"],
```
Comment (lilleyse) [inline Apps/Sandcastle/gallery/3D Tiles Feature Styling.html]: We had talked about this earlier, but is the cast required? Does it suggest something is different about the new tileset?

File: Apps/Sandcastle/gallery/3D Tiles Feature Styling.html [hunk 3]
```diff
@@ -105,14 +101,14 @@
 // Color buildings with a '3' in their name.
 function colorByNameRegex() {
     tileset.style = new Cesium.Cesium3DTileStyle({
-        color : "(regExp('3').test(${name})) ? color('cyan', 0.9) : color('purple', 0.1)"
+        color : "(regExp('3').test(String(${name}))) ? color('cyan', 0.9) : color('purple', 0.1)"
```
Comment (lilleyse) [inline Apps/Sandcastle/gallery/3D Tiles Feature Styling.html]: Is the string cast required?

**Comment by lilleyse** [comment]
We should also wait to merge until Draco is supported in IE: https://github.com/AnalyticalGraphicsInc/cesium/issues/6404.

**Comment by mramato** [comment]
Awesome.

What's the status of Draco support in IE11?  I assume this tileset won't load there?  We should probably hold off on merging this until IE11 works.

**Comment by hpinkos** [comment]
Marking this `next release` because it sounds like it can be merged as soon as #6626 is

**Comment by shehzan10** [comment]
I'm generating a new tileset that uses PBR materials instead of WebGL extension. Please wait for my word before merging this.

**Comment by shehzan10** [comment]
This is now ready for review and merge.

**Comment by hpinkos** [comment]
Looks great @shehzan10, thanks!

**Pull request merged**